Jenkinsfile

```diff
@@ -6,6 +6,9 @@ def pipeline = new org.rust.AppPipeline(steps: this,
       buildTestCmds: ['housekeeping/tests.sh'],
       cargoClippyTag: ':latest',
       cargoClippyCmds: ['housekeeping/clippy.sh'],
-      codeCoverage: false
+      codeCoverage: false,
```

Comment:
Why?

Reply:
There is no coverage script and env image does not have needed toosl as well.
I will need to discuss it with devs

Reply:
Discussed with Alexey - to add it we need to update project. He will create a task